prediction-polls/android/app/src/main/java/com/bounswe/predictionpolls/core/BaseRepository.kt

```diff
@@ -0,0 +1,31 @@
+package com.bounswe.predictionpolls.core
```

Comment:
will the base classes make testing harder for you?

Reply:
I don`t know I couldnt have enough time for unit tests. Will see in next tasks.